docs/src/main/sphinx/connector/influxdb.rst

```diff
@@ -0,0 +1,166 @@
+=======================
+InfluxDB connector
+=======================
+
+The InfluxDB Connector allows access to `InfluxDB <https://www.influxdata.com/>`_ data from Trino.
+This document describes how to setup the InfluxDB Connector to run SQL queries against InfluxDB.
+
+Requirements
+------------
+
+To connect to InfluxDB, you need:
+
+* InfluxDB v1.4.0 ~ v1.8.10
+* Network access from the Trino coordinator and workers to InfluxDB.
+
+Configuration
+-------------
+
+To configure the InfluxDB connector, create a catalog properties file
+``etc/catalog/example.properties`` with the following contents,
```

Comment:
Can we rename the file to `influxdb.properties`

Reply:
https://github.com/trinodb/trino/pull/15549#discussion_r1060231580